### PR TITLE
LPS-183326 ContentDashboardDropdownItemsProvider

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://issues.liferay.com/browse/LPS-183326)

##  Steps

1. Go to Control Menu > Content dashboard
2. Go down and see content section with dropdown 
Screenshot 2023-05-10 at 17 52 07

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java [hunk 1]
```diff
@@ -98,7 +81,20 @@ public List<DropdownItem> getDropdownItems(
...
+							"fetchURL",
+							ResourceURLBuilder.createResourceURL(
+								_liferayPortletResponse
+							).setBackURL(
+								_portal.getCurrentURL(_liferayPortletRequest)
+							).setParameter(
+								"className", infoItemReference.getClassName()
+							).setParameter(
+								"classPK",
+								String.valueOf(infoItemReference.getClassPK())
```
Comment (ealonso) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java:93]: @DiegoHu97 String.valueOf is not needed

**Comment by DiegoHu97** [comment]
@ealonso Updated 😄

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
@DiegoHu97, seems that the tests are related to the changes 😢

**Comment by DiegoHu97** [comment]
@ealonso  Updated! 😄

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by liulukiwi** [comment]
ci:forward

**Pull request closed without merging**